new-site/gatsby-ssr.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's SSR (Server Side Rendering) APIs in this file.
+ *
+ * See: https://www.gatsbyjs.com/docs/ssr-apis/
+ */
+
+// You can delete this file if you're not using it
```

Comment:
We could add this also later if we apply gatsby ssr

Reply:
- [ ] delete unused file